packages/react-devtools-shared/src/__tests__/profilingCache-test.js

```diff
@@ -630,6 +620,381 @@ describe('ProfilingCache', () => {
     }
   });
 
+  // @reactVersion > 18.2
+  // @gate !disableLegacyContext
+  it('should record changed props/state/context/hooks for React version (18.2; ∞) with legacy context enabled', () => {
```

Comment:
Let's hope we don't have legacy context enabled indefinitely 😅

Reply:
Hopefully this will be removed together with `disableLegacyContext` flag